csfieldguide/chapters/content/en/human-computer-interaction/sections/users-and-tasks.md

```diff
@@ -27,7 +27,7 @@ Another important thing to do when designing and evaluating an interface is to t
 Advertisements for digital devices often hide the complexity of a task, and simply point out the features available for doing the task.
 For example, suppose a smartphone is advertised as having a high resolution camera.
 The real task that someone might want to do is to take a photo of something they've just spotted, and send it to a friend.
-If you look at what happens in reality, the smartphone might be in their pocket or bag, and if they see something cool going past, they need to get it out, perhaps unlock it, open the camera app, adjust the lighting and other settings, press a button (is it easy to find when you're holding the camera up?), select the photo, choose a sharing option, select a friend to share it with (does the system help with that?), send it (what happens if you're out of reception range?), and then put the phone away.
+If you look at what happens in reality, the smartphone might be in their pocket or bag, and if they see something cool going past, they need to: get it out; perhaps unlock it; open the camera app; adjust the lighting and other settings; press a button (is it easy to find when you're holding the camera up?); select the photo; choose a sharing option; select a friend to share it with (does the system help with that?); send it (what happens if you're out of reception range?); and then put the phone away.
```

Comment:
Would this be better as a numbered list?

Reply:
I agree, will make the change

Reply:
Have done so